resources/zib/zib-DrugUse.xml

```diff
@@ -0,0 +1,230 @@
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueCodeableConcept">
+      <path value="Observation.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <short value="DrugUseStatus" />
+      <definition value="Indication stating whether there was any drug use in the past or present." />
+      <alias value="DrugsGebruikStatus" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.4.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.3" />
+        <comment value="DrugUseStatus" />
+      </mapping>
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Relevant comments on the drug use." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.9" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:amount">
+      <path value="Observation.component" />
+      <sliceName value="amount" />
+    </element>
+    <element id="Observation.component:amount.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="363908000" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:amount.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="Amount" />
+      <definition value="The number of units (pills, joints, shots etc.) per day, week, month or year or the frequency of use." />
+      <alias value="Hoeveelheid" />
+      <type>
+        <code value="string" />
+      </type>
+      <mapping>
+        <identity value="zib-druguse-v3.3-2020EN" />
+        <map value="NL-CM:7.4.8" />
+        <comment value="Amount" />
+      </mapping>
+    </element>
+    <element id="Observation.component:drugOrMedicationType">
+      <path value="Observation.component" />
+      <sliceName value="drugOrMedicationType" />
+    </element>
+    <element id="Observation.component:drugOrMedicationType.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="47429007" />
```

Comment:
I'm pretty sure this code is wrong. Let's wait for https://bits.nictiz.nl/browse/ZIB-1466 for the definite answer.

Reply:
Answer is 105590001

Reply:
Fixed